modules/helfi_address_search/src/Plugin/views/area/AddressSearchInfo.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\helfi_address_search\Plugin\views\area;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\views\Plugin\views\area\AreaPluginBase;
+
+/**
+ * Views area address_search_info handler.
+ *
+ * @ingroup views_area_handlers
+ *
+ * @ViewsArea("address_search_info")
+ */
+class AddressSearchInfo extends AreaPluginBase {
+
+  /**
+   * Get address search success status from the view.
+   *
+   * @return bool|null
+   *   TRUE: the search was successful
+   *   FALSE: address was not found
+   *   NULL: the search was not performed.
+   */
+  protected function getSearchStatus(): ?bool {
+    if (isset($this->view->element['address_search_succeed'])) {
+      return $this->view->element['address_search_succeed'];
+    }
+    return NULL;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function defineOptions() {
+    $options = parent::defineOptions();
+    $options['succeed'] = [
+      'contains' => [
+        'value' => ['default' => ''],
+        'format' => ['default' => NULL],
+      ],
+    ];
+    $options['failed'] = [
+      'contains' => [
+        'value' => ['default' => ''],
+        'format' => ['default' => NULL],
+      ],
+    ];
+    return $options;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildOptionsForm(&$form, FormStateInterface $form_state) {
```

Comment:
Missing return type (`void`) and `$form` is missing argument type

Reply:
Fixed.